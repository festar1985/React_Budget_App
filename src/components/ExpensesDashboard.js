import React from "react";
import ExpenseList from "./ExpenseList";
import ExpenseListFilters from "./ExpenseListFilters";
import ExpensesSummary from "./ExpensesSummary";

const ExpensesDashboard = () => (
  <div>
    <ExpensesSummary />
    <ExpenseListFilters />
    <ExpenseList></ExpenseList>
  </div>
);

export default ExpensesDashboard;
