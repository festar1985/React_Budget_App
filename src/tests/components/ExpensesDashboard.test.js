import React from "react";
import ExpensesDashboard from "../../components/ExpensesDashboard";
import { shallow } from "enzyme";
import toJSON from "enzyme-to-json";
import Adapter from "enzyme-adapter-react-16";
import Enzyme from "enzyme";

Enzyme.configure({ adapter: new Adapter() });

test("should render an expense list item when i give an example", () => {
  const wrapper = shallow(<ExpensesDashboard />);
  expect(toJSON(wrapper)).toMatchSnapshot();
});
