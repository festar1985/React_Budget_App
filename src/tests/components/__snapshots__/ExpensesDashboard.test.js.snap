// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render an expense list item when i give an example 1`] = `
<div>
  <Connect(ExpenseListFilters) />
  <Connect(ExpenseList) />
</div>
`;
